supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc

```diff
@@ -590,6 +590,28 @@ Use "Red Hat Virtualization Manager". Spell out in full for the first instance i
 
 *See also*: xref:zone-group[zone group]
 
+[discrete]
+[[regex]]
+==== image:images/no.png[no] regex (noun)
+*Description*: _Regex_ is an abbreviation for "regular expression". Do not use.
```

Comment:
I know it's in the "see also" section, but could you incorporate guidance to use "regular expression"? Maybe something like:

```suggestion
*Description*: _Regex_ is an abbreviation for "regular expression". Do not use; use "regular expression" instead.
```

or maybe even just:

```suggestion
*Description*: Do not use "regex" as a replacement for "regular expression".
```